### PR TITLE
Make port range larger in test

### PR DESCRIPTION
**Pull request by mortalisk**
Test was not working when many workers running, as no ports were available.
https://ci.equinor.com/scout/job/komodo_testing/job/ert-custom-script-pr/9340/console

### CODE REVIEW

File: ert3/evaluator/_evaluator.py [hunk 1]
```diff
@@ -50,8 +50,9 @@ def _run(
 
 def evaluate(
     ensemble: Ensemble,
+    custom_port_range: Optional[range] = None,
```
Comment (xjules) [inline ert3/evaluator/_evaluator.py:53]: Are we to expose `port_range` as parameter here?

Comment (mortalisk) [reply ert3/evaluator/_evaluator.py:53]: I dunno. Any reason why not?

Comment (xjules) [reply ert3/evaluator/_evaluator.py:53]: You might be right though as I can't see any other way to deliver the port range there.

Comment (jondequinor) [reply ert3/evaluator/_evaluator.py:53]: I agree with both of you.

Comment (mortalisk) [reply ert3/evaluator/_evaluator.py:53]: :monkey: :monkey: :monkey: :monkey: :question:

**Review by xjules — APPROVED** [review]
🚀

**Pull request merged**